src/Faker/Provider/Internet.php

```diff
@@ -5,7 +5,7 @@
 class Internet extends Base
 {
     protected static $freeEmailDomain = array('gmail.com', 'yahoo.com', 'hotmail.com');
-    protected static $tld = array('com', 'com', 'com', 'com', 'com', 'com', 'biz', 'info', 'net', 'org');
```

Comment:
Not even sure why there are duplicates - is there some reason I'm unaware of, @fzaninotto?

Reply:
Just thinking loud... could it be, that this helps bringing some things up more often by the faker? I mean, if you fake a domain it’s 40/50/60% a toplevel domain with .com? That would make sense to me, same for name formats which are more common in some situations/countries

Reply:
indeed, the repetition is intended